### PR TITLE
Feature/composite metrics

### PR DESCRIPTION
**Pull request by drio**

### CODE REVIEW

**Comment by obfuscurity** [comment]
:heart::+1::zap:

**Comment by jderrett** [comment]
:+1:

File: librato/__init__.py [hunk 1]
```diff
@@ -236,8 +236,13 @@ def get_composite(self, compose, **query_props):
...
         return self._mexe("metrics", method="GET", query_props=query_props)
 
+    def create_composite(self, name, compose, **query_props):
+        query_props['composite'] = compose
+        query_props['type'] = 'composite'
+        return self.update(name, **query_props)
+
     def update(self, name, **query_props):
-        resp = self._mexe("metrics/%s" % self.sanitize(name), method="PUT", query_props=query_props)
+        return self._mexe("metrics/%s" % self.sanitize(name), method="PUT", query_props=query_props)
```
Comment (jderrett) [inline librato/__init__.py]: Nice find :-)

Comment (drio) [inline librato/__init__.py]: Thanks!

**Pull request merged**